packages/firebase_database/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Flutter plugin for Firebase Database, a cloud-hosted NoSQL database
   with realtime data syncing across Android and iOS clients, and offline access.
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/firebase_database
-version: 3.0.0
+version: 3.0.1
```

Comment:
Maybe this should be `3.0.0+1` to match the version bumps in the other plugin packages.

Reply:
We're following the versioning scheme as described here: https://dart.dev/tools/pub/versioning#semantic-versions

For packages with version number smaller than 1.0.0 the minor number is after the `+` (e.g the scheme is `X.Y.Z` if X>=1, `0.X.Y+Z` else)